s3tests_boto3/functional/test_s3.py

```diff
@@ -7797,7 +7797,7 @@ def test_set_bucket_tagging():
     e = assert_raises(ClientError, client.get_bucket_tagging, Bucket=bucket_name)
     status, error_code = _get_status_and_error_code(e.response)
     eq(status, 404)
-    eq(error_code, 'NoSuchTagSet')
+    eq(error_code, 'NoSuchTagSetError')
```

Comment:
`NoSuchTagSetError` is not the correct error code. this was fixed for https://tracker.ceph.com/issues/55460, and the s3tests were changed to use the correct `NoSuchTagSet` in https://github.com/ceph/s3-tests/pull/452. that change is present on the ceph-quincy branch: https://github.com/ceph/s3-tests/commit/722c2a90bf08dc5a4110fd83f756faef7ed855bb

according to the quincy backport https://tracker.ceph.com/issues/55614, this was fixed in quincy v17.2.7. so earlier point releases are expected to fail on this test